scripts/agent-trader/agent-trader-prompt.md

```
@@ -31,18 +31,18 @@ Working dir: `scripts/agent-trader/`. The harness is `agent_trader.py`. Bets log
-   m = requests.get(f"https://gamma-api.polymarket.com/markets/{MID}").json()
-   p = json.loads(m["outcomePrices"])
-   mkt = {"market_id": str(MID), "slug": m["slug"], "question": m["question"],
-          "end_date": m["endDate"], "yes_price": float(p[0]),
-          "best_bid": float(m["bestBid"]), "best_ask": float(m["bestAsk"]),
-          "spread": float(m["spread"])}
-   record_bet(mkt, "YES_or_NO", p_hat_yes, "rationale...", stake=25.0, confidence="...")
+   (aim ≥ ~0.05 net edge; never bet a view you can't defend). Watch concentration — the
+   veto keys on the underlying real-world outcome, NOT on the ticker or the expiry date:
+   two markets settled by the same event, the same electorate on the same night, or the
+   same variable at a later date are ONE bet for this purpose. Record each via:
+   ```bash
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)